react/Section.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from 'react'
+import { useCssHandles, applyModifiers } from 'vtex.css-handles'
+import { useResponsiveValue } from 'vtex.responsive-values'
+
+interface Props {
+  borderless?: boolean
+  marginBottom?: number
+  paddingBottom?: number
+  width?: string | { mobile: string; desktop: string }
```

Comment:
⛏ [Nitpick] `vtex.responsive-values` exports `ResponsiveInput<T>` and `MaybeResponsiveInput<T>` types. In this case you could use `width?: MaybeResponsiveInput<string>`.

Reply:
It doesn't 😢  

![image](https://user-images.githubusercontent.com/12702016/72078457-e59f4800-32d7-11ea-8f02-3bc553798d7d.png)

Reply:
That's sad :(